prometheus/process_collector_other.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2015 The Prometheus Authors
```

Comment:
Not that it matters a lot, but in a new file, we usually update the year to the current one.